backend/utils.js

```diff
@@ -159,6 +159,12 @@ const ensureIsArray = (data) => (Array.isArray(data) ? data : [data])
 
 const maybePluralise = (noun, count, suffix = 's') => `${count} ${noun}${count !== 1 ? suffix : ''}`
 
+const possessive = (string) => {
+  if (!string) return ''
+
+  return `${string}${string.toLowerCase().endsWith('s') ? '’' : '’s'}`
```

Comment:
I have to ask - are you sure this is the correct apostrophy? I have seen Jamie use an html encoding to get the right one before.

Reply:
It's ok in this instance to use the character and is easier to read.  It's the same function copied over from PSH.